<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeletePopup</class>
 <widget class="QWidget" name="DeletePopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete Event</string>
  </property>
  <widget class="QDialogButtonBox" name="deleteEventButtons">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>191</width>
     <height>39</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::No|QDialogButtonBox::Yes</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="deleteEventQuestionText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>521</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Are you sure you wish to delete this event?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>deleteEventQuestionText</zorder>
  <zorder>deleteEventButtons</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
